package.json

```diff
@@ -39,7 +40,7 @@
     "moment": "^2.29.3",
     "mustache": "^4.2.0",
     "newman": "^6.0.0",
-    "path-to-regexp": "^6.2.1",
+    "path-to-regexp": "6.2.1",
```

Comment:
Temporarily pin path-to-regexp, upgrade in separate PR and react to breaking changes